lib/build_targets/package.dart

```diff
@@ -323,35 +334,25 @@ class NativeTpk {
       );
     }
 
-    final String tpkArch = buildInfo.targetArch
-        .replaceFirst('arm64', 'aarch64')
-        .replaceFirst('x86', 'i586');
-    final File outputTpk = buildDir.childFile(
-        tizenProject.outputTpkName.replaceFirst('.tpk', '-$tpkArch.tpk'));
-    if (!outputTpk.existsSync()) {
+    // TODO(pkosko): find better way to determine file name
+    File outputTpk;
+    for (final File file in buildDir.listSync().whereType<File>()) {
+      if (file.basename.endsWith('.tpk')) {
+        outputTpk = file;
+        break;
+      }
+    }
+    if (outputTpk == null || !outputTpk.existsSync()) {
       throwToolExit(
-          'Build succeeded but the expected TPK not found:\n${result.stdout}');
+          'Build succeeded but the expected TPK ($outputTpk) not found:\n${result.toString()}');
     }
-
     // Copy and rename the output TPK.
     outputTpk.copySync(outputDir.childFile(tizenProject.outputTpkName).path);
+    environment.logger.printStatus(
+        'Generated tpk file was moved to : ${outputDir.childFile(tizenProject.outputTpkName).path}');
```

Comment:
If you apply my suggestion in the above, this message becomes no longer informative for users. Can be removed or at least `tpk` should be capitalized.